extension/src/pipeline/index.ts

```diff
@@ -83,28 +83,8 @@ export class Pipeline extends DeferredDisposable {
   }
 
   public async getCwd() {
-    const focusedPipeline = this.getFocusedPipeline()
-    if (focusedPipeline) {
-      return focusedPipeline
-    }
-
     await this.checkOrAddPipeline()
-
-    const pipelines = this.model.getPipelines()
-    if (!pipelines?.size) {
-      return
-    }
-    if (pipelines.has(this.dvcRoot)) {
-      return this.dvcRoot
-    }
-    if (pipelines.size === 1) {
-      return [...pipelines][0]
-    }
-
-    return quickPickOne(
-      [...pipelines],
-      'Select a Pipeline to Run Command Against'
-    )
+    return this.findPipelineCwd()
```

Comment:
Going to mention it here since it was discussed asynchronously with Matt, but it would simplify things to just have the Plot Wizard automatically create a dvc.yaml in the root vs looking for one elsewhere. 

While this means that it wouldn't work for people with workspaces or nested dvc.yamls, users who do those are most likely advanced DVC users who don't have a need for the plot wizard. 

I chose to keep it like it is for now but I'm happy to update if preferred.